var/spack/repos/builtin/packages/libspng/package.py

```diff
@@ -14,10 +14,18 @@ class Libspng(CMakePackage):
     homepage = "https://github.com/randy408/libspng"
     url = "https://github.com/randy408/libspng/archive/refs/tags/v0.7.2.tar.gz"
 
+    maintainers("sethrj")
     license("BSD-2-Clause")
```

Comment:
```suggestion
    maintainers("sethrj")

    license("BSD-2-Clause")
```

Quick suggestion to add a space between `maintainers()` and `license()` directives as described in the packaging guide.